lib/src/main/java/org/team100/lib/motor/drive/Falcon6DriveMotor.java

```diff
@@ -26,54 +26,43 @@
  * Phoenix 6 uses a Kalman filter to eliminate velocity measurement lag.
  */
 public class Falcon6DriveMotor implements MotorWithEncoder100<Distance100> {
-
+    //TODO Tune ff for amps
     /**
      * The speed, below which, static friction applies, in motor revolutions per
      * second.
      */
     private static final double staticFrictionSpeedLimitRev_S = 3.5;
 
     /**
-     * Friction feedforward in volts, for when the mechanism is stopped, or nearly
+     * Friction feedforward in amps, for when the mechanism is stopped, or nearly
      * so.
      */
-    private static final double staticFrictionFFVolts = 0.18;
+    private static final double staticFrictionFFAmps = 0.18;
 
     /**
-     * Friction feedforward in volts, for when the mechanism is moving.
+     * Friction feedforward in amps, for when the mechanism is moving.
      * 
      * This value seems very low, perhaps because the falcon closed-loop control is
      * compensating?
      */
-    private static final double dynamicFrictionFFVolts = 0.01;
+    private static final double dynamicFrictionFFAmps = 0.01;
 
     /**
-     * Velocity feedforward in units of volts per motor revolution per second, or
-     * volt-seconds per revolution. Since saturation is 11 volts and free speed is
+     * Velocity feedforward in units of amps per motor revolution per second, or
+     * amp-seconds per revolution. Since saturation is 11 amps and free speed is
```

Comment:
you can't just search and replace the word "volts" without changing the meaning of this sentence.